pango/src/auto/layout.rs

```diff
@@ -421,6 +427,18 @@ impl Layout {
         }
     }
 
+    #[cfg(any(feature = "v1_50", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_50")))]
+    #[doc(alias = "pango_layout_serialize")]
+    pub fn serialize(&self, flags: LayoutSerializeFlags) -> Option<glib::Bytes> {
```

Comment:
And neither this

Reply:
All of those are caused by trust_nullability :/

Reply:
Yes, if we opt in to that we need to keep the upstream annotations correct